testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/MixedDomainTestSupport.java

```diff
@@ -174,16 +183,15 @@ private void configureSecondaryJavaHome() {
 
     private void startAndAdjust() {
 
-        String jbossDomainServerArgsValue = null;
         try {
-            //Start the primary in admin only  and reconfigure the domain with what
+            //Start the primary in admin only and reconfigure the domain with what
             //we want to test in the mixed domain and have the DomainAdjuster
             //strip down the domain model to something more workable. The domain
             //adjusters will also make adjustments for the legacy version being
             //tested.
             DomainLifecycleUtil primaryUtil = getDomainPrimaryLifecycleUtil();
+            assert primaryUtil.getConfiguration().getStability() == version.getStability();
             primaryUtil.getConfiguration().setAdminOnly(true);
-            //primaryUtil.getConfiguration().addHostCommandLineProperty("-agentlib:jdwp=transport=dt_socket,address=8787,server=y,suspend=y");
```

Comment:
BTW I think this was intentionally left out here for connecting a remote debugger since failures here are pretty hard to investigate otherwise.

Reply:
Hi @rhusar , yes, I replaced it by using environment variables, you can now the DC with `mvn clean .... -Dwildfly.primary.debug` and the legacy host with "-Dwildfly.secondary.debug"

Reply:
@yersan Excellent, even better! Thanks.